backend/functions/lib/configs/application-configuration.ts

```diff
@@ -78,11 +79,12 @@ export class ApplicationConfiguration {
     return config;
   }
 
-  private getCosmosConfig(): CosmosConfig {
+  private getCosmosConfig(): DocumentDbConfig {
```

Comment:
Refactor cosmos out of function name.
```suggestion
  private getDocumentDbConfig(): DocumentDbConfig {
```